OpenSourceLicenses.md

```diff
@@ -9,5 +9,7 @@ The binary distributions are directly from the `gh-pages` branch.
 
 The bootstrap template is from:
 https://github.com/StartBootstrap/startbootstrap-bare.git
-with a modified contents, modified navbar
-and theming based on the original web page
+which is also available under an MIT license.
+
+This repo includes modified content, modified navbar
+and theming based on the original web page.
```

Comment:
This is a change that I made to main 9 days ago.
https://github.com/e-mission/em-public-dashboard/commit/893fd0a3a5e6a9ccded82d1793459a25450d30f8

Not sure why it is showing up as a change to your branch. Did you not pull from main before making your change?